code/ATMOSPHERICS/components/unary/cap.dm

```diff
@@ -125,3 +125,105 @@
 		return // Coloring pipes.
 
 	return ..()
+
+/obj/machinery/atmospherics/unary/cap/bluespace
+	name = "bluespace pipe endcap"
+	desc = "A bluespace endcap for pipes."
```

Comment:
This description doesn't say a lot for the function of the device, especially since it's not really obvious at first glance.